config/default.yml

```diff
@@ -45,6 +45,12 @@ Minitest/AssertTruthy:
   Enabled: true
   VersionAdded: '0.2'
 
+Minitest/AssertEmptyLiteral:
+  Description: 'This cop enforces the test to use `assert_empty` instead of using `assert([], object)` or `assert({}, object)`.'
+  StyleGuide: 'https://github.com/rubocop-hq/minitest-style-guide#assert-empty-literal'
```

Comment:
I think this is a cop that acts as a Linter to prevent bugs. So I probably don't need a style guide.

Reply:
removed